### PR TITLE
fix: resend notification when case state not Open or Deleted [FPLA-NA]

### PR DESCRIPTION
**Pull request by Gogetter**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RoboticsController.java [hunk 1]
```diff
@@ -37,9 +41,11 @@ private void performVerification(final String caseId, final CaseDetails caseDeta
             throw new ResponseStatusException(HttpStatus.NOT_FOUND, String.format("No case found with id %s", caseId));
         }
 
-        if (!caseDetails.getState().equals("Submitted")) {
+        List<String> donNotSendStates = asList("Open", "Deleted");
```
Comment (tomaszpowroznik) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RoboticsController.java:44]: can be static, List.of is preferable (produces immutable list which should be used here)

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RoboticsController.java [hunk 2]
```diff
@@ -14,11 +14,17 @@
...
+import java.util.List;
+
+import static java.util.List.of;
+
 @Api
 @RestController
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
 @ConditionalOnProperty(prefix = "feature.toggle", name = "robotics.support.api.enabled", havingValue = "true")
 public class RoboticsController {
+    private static final List<String> doNotSendStates = of("Open", "Deleted");
```
Comment (tomaszpowroznik) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RoboticsController.java:26]: should be capitaised

Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RoboticsController.java:26]: There is a state enum, could this be used?

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RoboticsController.java [hunk 3]
```diff
@@ -37,9 +45,10 @@ private void performVerification(final String caseId, final CaseDetails caseDeta
             throw new ResponseStatusException(HttpStatus.NOT_FOUND, String.format("No case found with id %s", caseId));
         }
 
-        if (!caseDetails.getState().equals("Submitted")) {
+        if (EXCLUDED_STATES.contains(caseDetails.getState())) {
             throw new ResponseStatusException(HttpStatus.BAD_REQUEST,
-                String.format("Unable to proceed as case  with id %s has not been submitted", caseId));
+                String.format("Unable to proceed as case  with id %s is the the wrong state", caseId));
```
Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RoboticsController.java:50]: Do we need to add tests to cover this updated scenario

**Pull request merged**